doc/filters/format_currency.rst

```diff
@@ -74,4 +134,16 @@ Arguments
 
 * ``currency``: The currency
 * ``attrs``: A map of attributes
-* ``locale``: The locale
+* ``locale``: The locale (only 'en' supported at the moment)
```

Comment:
this is not true. If you install `ext-intl`, all locales are supported. The restriction of only `en` does not come from the Twig filter. It comes from `symfony/polyfill-intl-icu` which polyfills the `NumberFormatter` but does not have the formatting rules for hundreds of locales.